components/layout/construct.rs

```diff
@@ -1842,17 +1822,17 @@ fn bidi_control_chars(style: &Arc<ServoComputedValues>) -> Option<(&'static str,
 }
 
 fn control_chars_to_fragment(node: &InlineFragmentNodeInfo,
+                             context: &SharedStyleContext,
                              text: &str,
                              restyle_damage: RestyleDamage)
                              -> Fragment {
     let info = SpecificFragmentInfo::UnscannedText(
         box UnscannedTextFragmentInfo::new(String::from(text), None));
-    let mut style = node.style.clone();
-    properties::modify_style_for_replaced_content(&mut style);
-    properties::modify_style_for_text(&mut style);
+    let text_style = context.stylist.style_for_anonymous_box(
```

Comment:
The name of the method is still `style_for_anonymous_box`, which is no longer true.